workflow/artifacts/raw/raw.go

```diff
@@ -0,0 +1,26 @@
+package raw
+
+import (
+	"github.com/argoproj/argo/errors"
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"os"
+)
+
+type RawArtifactDriver struct {
+}
+
+// Store raw content as artifact
+func (a *RawArtifactDriver) Load(artifact *wfv1.Artifact, path string) error {
+	lf, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	defer lf.Close()
```

Comment:
This will fail linting since the return value is unchecked. You will need to put it an anonymous function which explicitly ignores the value using _ = lf.Close()

Reply:
good point. will make the change. not sure why lint didn't catch this